MAVProxy/modules/mavproxy_link.py

```diff
@@ -675,6 +675,19 @@ def heartbeat_is_from_autopilot(self, m):
 
         return True
 
+    mav_type_planes = [
+        mavutil.mavlink.MAV_TYPE_FIXED_WING,
+        mavutil.mavlink.MAV_TYPE_VTOL_QUADROTOR,
+        mavutil.mavlink.MAV_TYPE_VTOL_TILTROTOR,
+    ]
+    # VTOL_DUOROTOR was renamed to VTOL_TAILSITTER_DUOROTOR
+    for possible_plane_type in "VTOL_DUOROTOR", "VTOL_TAILSITTER_DUOROTOR":
```

Comment:
You should never get "VTOL_DUOROTOR" after update because it isn't present in the mavlink XML and what is received will be a number. Perhaps I'm missing something.